modules/store.js

```diff
@@ -0,0 +1,33 @@
+// Store Class: Handles Storage
+class Store {
+  static getBooks() {
```

Comment:
- [ ] Kindly make all the functions in this class to be arrow function = () =>. This would show that you are updated as regards to ES6.